.github/workflows/build_and_test_dbt.yaml

```diff
@@ -6,6 +6,16 @@ on:
   push:
     branches: [master]
 
+  workflow_dispatch:
+    inputs:
+      models:
+        required: true
+        description: >
+          A space-separated list of dbt models to build.
+          Prefix with the + sign to rebuild upstream models
```

Comment:
**[Suggestion, non-blocking]** Do you feel confident that everyone on the team will understand the proper format for a dbt model (particularly, that the schema is required)? If not, perhaps we should include an example?

```suggestion
          A space-separated list of dbt models to build.
          Prefix with the + sign to rebuild upstream models.
          For example, "default.vw_pin_sale default.vw_pin_universe".
```